index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile First Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <style> @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); </style>
+  <style> @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); </style>
+  <style> @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); </style>
+</head>
+<body>
+  <header>
+    <div class="toolbar">
+      <a href="https://github.com/bilalrajput09" class="name">
+        Bilal Ahmed
+      </a>
```

Comment:
- [ ] I see 😊 that the elements not started from the left top corner this happened because of the default margin and padding of elements, so I see 😃 you didn't put the global style for elements, so kindly fix this by adding the global style using the ```*``` mark and assigning to it 👌
``` margin: 0; padding: 0; box-sizing: border-box;```

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/212329627-920a9174-ad66-4a48-bed9-8030b835cd60.png) |    ![image](https://user-images.githubusercontent.com/83360636/212329536-227548dc-785b-4695-b495-4c73424e3225.png) |